rasa_nlu/featurizers/count_vectors_featurizer.py

```diff
@@ -146,13 +146,17 @@ def process(self, message, **kwargs):
                          "component is either not trained or "
                          "didn't receive enough training data")
         else:
-            bag = self.vect.transform([self._lemmatize(message)]).toarray()
+            bag = self.vect.transform(
+                [self._get_message_text(message)]
+            ).toarray()
             message.set("text_features", bag)
 
     @staticmethod
-    def _lemmatize(message):
-        if message.get("spacy_doc"):
+    def _get_message_text(message):
+        if message.get("spacy_doc"):  # if lemmatize is possible
             return ' '.join([t.lemma_ for t in message.get("spacy_doc")])
+        elif message.get("tokens"):  # if directly tokens is provided
+            return ' '.join(message.get("tokens"))
```

Comment:
Could you please add a test that generated tokens satisfy provided `token_pattern` , because `CountVectorizer` performs tokenization, by finding all expressions that fall under regex with provided `token_pattern` like here: https://github.com/RasaHQ/rasa_nlu/blob/59d2d15a127f4e6892380cc95242b8151d6621df/rasa_nlu/featurizers/count_vectors_featurizer.py#L147

Reply:
Sure, I will add some test codes for this.